core/utils/hash_helpers_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/assert"
 
-	"github.com/smartcontractkit/chainlink/v2/core/utils"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils"
 )
 
 func TestNewHash(t *testing.T) {
```

Comment:
Since these tests are using methods from the `core/chains/evm/utils`, should we move these to the utils_test in that directory?